solutions/swb-reference/integration-tests/support/clientSession.ts

```diff
@@ -99,6 +101,17 @@ export default class ClientSession {
   public getSetup(): Setup {
     return this._setup;
   }
+
+  public getUserId(): string | undefined {
```

Comment:
what about defining the property as `public readonly userId?: string` so we don't need to write the getter manually?

Reply:
This class already has `getAxiosInstance`, `getSettings`, `getSetup`. I'd keep it consistent, but good call for best practices - we should prefer one over another

Reply:
quick refactor 🤷  lol jk sounds good